services/web-server/.neutrinorc.js

```diff
@@ -0,0 +1,36 @@
+const nodeExternals = require('webpack-node-externals');
+
+module.exports = {
+  use: [
+    ['@neutrinojs/node', {
+      hot: false,
+      babel: {
+        plugins: [
+          [require.resolve('@babel/plugin-proposal-decorators'), { legacy: true }],
+          require.resolve('@babel/plugin-proposal-class-properties'),
+        ]
+      },
+    }],
+    (neutrino) => {
+      neutrino.config.merge({
+        externals: [
+          nodeExternals({
+            whitelist: [/^webpack/]
+          }),
+          nodeExternals({
+            modulesFromFile: true,
+          }),
+          nodeExternals({
+            modulesDir: '../../node_modules',
```

Comment:
Using forward slashes might cause trouble for users running on windows. Another way to write this could be with:

```jsx
const { join } = require('path');

// ...

modulesDir: join('..', '..', 'node_modules');
```

Reply:
oh cool!